appshared/src/main/java/io/reark/rxgithubapp/shared/view/RepositoriesView.java

```diff
@@ -76,7 +83,7 @@ protected void onFinishInflate() {
         super.onFinishInflate();
 
         EditText editText = (EditText) findViewById(R.id.repositories_search);
-        searchStringObservable = RxTextView.textChanges(editText).map(CharSequence::toString);
+        searchStringObservable = RxTextView.textChanges(editText).map(CharSequence::toString).toFlowable(BackpressureStrategy.BUFFER);
```

Comment:
@ulx, I think you should change all Flowables to Observables by default in this PR. We can in a separate change check which of them should be Flowables, Singles or Maybes.